jumanji/environments/logic/minesweeper/constants.py

```diff
@@ -16,8 +16,9 @@
 IS_MINE: int = 1
 PATCH_SIZE: int = 3
 REVEALED_EMPTY_SQUARE_REWARD: float = 1.0
-REVEALED_MINE_OR_INVALID_ACTION_REWARD: float = 0.0
-COLOUR_MAPPING: list = [
+REVEALED_MINE_REWARD: float = 0.0
+INVALID_ACTION_REWARD: float = 0.0
```

Comment:
Could we remove these globals and have them defined in the reward function with these defaults values used to instantiate the function in the environment init?

Reply:
Also includes the one above: `REVEALED_EMPTY_SQUARE_REWARD`

Reply:
OK, but why?